src/VSCodeExtension/src/formatter/rules/operation.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
I prefer the original structure where we expose only an array of rules. So, this `operation.ts` file would expose a set of `rules` that includes `argsRule`. We would then import them and unroll them out, so that we don't clutter with the potentially huge number of rules.